zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -542,6 +542,66 @@ paths:
             application/problem+json:
               schema:
                 $ref: "#/components/schemas/ProblemDetail"
+  /user-tasks/{userTaskKey}/form:
+    get:
+      tags:
+        - User Task
+      summary: Return user task form
```

Comment:
I think we need a period at the end.

Reply:
No, we prefer no period at the end of the endpoint `summary` property. This line meets our recommendations without changes. 

See item 3.iv in https://github.com/camunda/camunda/blob/main/docs/rest-controller.md#openapi-extension for casing recommendations. We don't want periods on the `summary` because it is used in the left navigation of the docs: 

<img width="303" alt="image" src="https://github.com/user-attachments/assets/612e1a12-ba1f-4a22-8f6f-6eac36270f46">